The "build-and-push-image" job from the release workflow fails often when running npm for ARM builds

The job fails exceptionally often for the ARM64 image build:

```
------
 > [linux/arm64 ui  9/11] RUN make node_modules:
 ERR! network Socket timeout
#52 805.0 npm ERR! network This is a problem related to network connectivity.
#52 805.0 npm ERR! network In most cases you are behind a proxy or have bad network settings.
#52 805.0 npm ERR! network 
#52 805.0 npm ERR! network If you are behind a proxy, please make sure that the
#52 805.1 npm ERR! network 'proxy' config is set properly.  See: 'npm help config'
#52 809.2 
#52 809.2 npm ERR! A complete log of this run can be found in:
#52 809.2 npm ERR!     /home/node/.npm/_logs/2023-01-04T16_22_54_895Z-debug-0.log
#52 810.3 make: *** [Makefile:165: node_modules] Error 1
------
gitops-server.dockerfile:11
--------------------
   9 |     COPY --chown=node:node tsconfig.json /home/app/
  10 |     COPY --chown=node:node .parcelrc /home/app/
  11 | >>> RUN make node_modules
  12 |     COPY --chown=node:node ui /home/app/ui
  13 |     RUN --mount=type=cache,target=/home/app/ui/.parcel-cache make ui
--------------------
ERROR: failed to solve: process "/bin/sh -c make node_modules" did not complete successfully: exit code: 2
```

[Latest occurrence](https://github.com/weaveworks/weave-gitops/actions/runs/3839600120/attempts/3)

--- a/.npmrc
+++ b/.npmrc
@@ -1,1 +1,2 @@
 @weaveworks:registry=https://npm.pkg.github.com
+fetch-timeout=1500000

--- a/gitops-server.dockerfile
+++ b/gitops-server.dockerfile
@@ -8,6 +8,7 @@ COPY --chown=node:node package*.json /home/app/
 COPY --chown=node:node Makefile /home/app/
 COPY --chown=node:node tsconfig.json /home/app/
 COPY --chown=node:node .parcelrc /home/app/
+COPY --chown=node:node .npmrc /home/app/
 RUN make node_modules
 COPY --chown=node:node ui /home/app/ui
 RUN --mount=type=cache,target=/home/app/ui/.parcel-cache make ui

